### DISCUSSION TITLE
Feature of adding play directly from user codebase through website

### DISCUSSION THREAD

**Discussion by shrilakshmishastry** [reactplay/react-play#116] [Ideas]
### Current behavior:
1. User needs to clone the react-play repo
2.  follow cmd steps to create play
3. copy-paste from their original play / create one
4. push it back to react-play codebase at GitHub

### Feature Request 
1. Feature in the website itself, where the user pastes his play codebase URL which is in GitHub 
2.  fetch code from his codebase and add play automatically

**Comment by atapas** [comment]
Interesting, how would the code review phase we are going to satisfy?

**Reply by shrilakshmishastry** [reply]
Can we add an intermediate stage, like once the play is taken, it automatically creates a branch in GHU, and the code review phase can be done and then merged?

**Reply by atapas** [reply]
Yes, I would love to try that automation. This is something we need to explore.

Something like,

- Create a branch automatically
- The branch should have all the skeleton code we creating already
- Create an issue for the branch work
- Assign the issue to the user who created the play.

I'm not sure yet, how much of this workflow automation is possible today. It is worth exploring.

**Reply by atapas** [reply]
@koustov is doing the design for a create workflow from the UI itself.

**Comment by MOHAMED-EHAB-DEV** [comment]
Very Interesting
